dev-vcs/guitar/guitar-0.2.1~git.recipe

```diff
@@ -0,0 +1,85 @@
+SUMMARY="A QT git client"
+DESCRIPTION="Git GUI Client
+
+* Open Source, Free Software
+* Run on Windows, macOS and Linux
+* Written in C++
+* Powered by Qt 5"
+HOMEPAGE="https://soramimi.github.io/Guitar/"
+COPYRIGHT="2017 S.Fuchita"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="828a6f934d86f8eb42a81c12fdb6ae52063a6835"
+SOURCE_URI="https://github.com/soramimi/Guitar/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="dd6d5f54856dbb6911993344bad2e684564654a04ddf01ee1e016d7018654178"
+SOURCE_DIR="Guitar-$srcGitRev"
+SOURCE_FILENAME="Guitar-$portVersion-$srcGitRev.tar.gz"
+ADDITIONAL_FILES="guitar.rdef.in"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64 ?arm ?ppc"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	guitar$secondaryArchSuffix = $portVersion
+	app:guitar
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	cmd:git
```

Comment:
The app builds and seems to works fine. Tested on x86_64 and x86 second arch.

```diff
 REQUIRES="
 	haiku$secondaryArchSuffix
+	cmd:file
 	cmd:git
```
This is because the app asks for the paths of both the `git` and `file` commands.

BTW, I'd also recommend this change in `PROViDES`:
```diff
-	app:guitar
+	app:Guitar
```